lib/db_query_matchers/version.rb

```diff
@@ -1,4 +1,4 @@
 # Defines the gem version.
 module DBQueryMatchers
-  VERSION = '0.11.0'
+  VERSION = '1.0.0'
```

Comment:
Wouldn't this be more like a `0.11.1` release than a major jump?

Reply:
Yes, actually you are right